qoffer-domain/src/main/groovy/life/qbic/portal/portlet/offers/create/CalculatePrice.groovy

```diff
@@ -0,0 +1,23 @@
+package life.qbic.portal.portlet.offers.create
+
+import life.qbic.datamodel.dtos.business.AffiliationCategory
+import life.qbic.datamodel.dtos.business.ProductItem
+
+/**
+ * An interface handling the calculation of a price for an offer
+ *
+ * Calculates the price for an offer
+ *
+ * @since: 0.1.0
```

Comment:
```suggestion
 * @since: 1.0.0
```

Reply:
1.0.0 will be the first official release

Reply:
Ok, thought I we said that we want to make pre-releases with 0.1.0. But I can change that back